pkg/kubectl/cmd/set/set_serviceaccount.go

```diff
@@ -36,14 +36,11 @@ import (
 )
 
 var (
-	serviceaccountResources = `
-	replicationcontroller (rc), deployment (deploy), daemonset (ds), job, replicaset (rs), statefulset`
-
```

Comment:
Note to reviewers: This variable is not used anywhere except for line 46. Remove this variable because it is blocking the generation of file template.pot.

```
$ hack/update-translations.sh -x
Extracting strings to POT
panic: unknown type: serviceaccountResources
...
```